### PR TITLE
Workflow/pytorch new release

### PR DESCRIPTION
**Pull request by ClaudiaComito**
## Description

This workflow is made up of 2 workflows:

1. `.github/workflows/pytorch-latest-release.yml` is a cron job running at 0500 UTC on workdays (can only be tested on `main`). It checks for pytorch's latest release tag, and if it's different from the stored one, it commits and pushes the latest tag.  The `git push` from this workflow triggers the next one.
2. `.github/workflows/latest-pytorch-support.yml`:
 - creates an Issue "support latest pytorch release"
 - branching off `main`, modifies `setup.py` to extend the range of allowed pytorch versions to the latest one
 - creates a PR with the latest `setup.py` and runs the CPU tests (see #932)

This workflow does not:
- merge to `main` or anywhere
- trigger GPU tests on our cluster. 

Issue/s resolved: #

## Type of change

CI/CD

## Memory requirements

NA
## Performance

NA

## Due Diligence
- [ ] All split configurations tested
- [ ] Multiple dtypes tested in relevant functions
- [ ] Documentation updated (if needed)
- [ ] Updated changelog.md under the title "Pending Additions"

#### Does this change modify the behaviour of other functions? If so, which?
no

### CODE REVIEW

File: .github/workflows/pytorch-latest-release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          jq -r ".tag_name" | tr -d 'v' > .github/workflows/pytorch-release-versions/pytorch-latest.txt
+      - name: Check for modified files
+        id: git-check
+        run: echo ::set-output name=modified::$([ -z "`git status --porcelain`" ] && echo "false" || echo "true")
+      - name: Commit latest PyTorch release version
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          git config --global user.name 'ClaudiaComito'
+          git config --global user.email 'c.comito@fz-juelich.de@users.noreply.github.com'
+          git commit -am "New PyTorch release"
```
Comment (bhagemeier) [inline .github/workflows/pytorch-latest-release.yml:25]: Should we add the actual release version here, just to be precise?

**Review by bhagemeier — APPROVED** [review]
Looks awesome. There's one minor comment I made at the corresponding line. Generally, the awk, sed, tr magic seems a bit convoluted, but there's probably no alternative. I have not tested this workflow, but trust that you've done your tests of it.

**Comment by ClaudiaComito** [comment]
run tests

File: .github/workflows/pytorch-latest-release.yml (hunk 1 above)
Comment (ClaudiaComito) [reply .github/workflows/pytorch-latest-release.yml]: Done. Also added release version to PR title

**Comment by ClaudiaComito** [comment]
> 



> Looks awesome. There's one minor comment I made at the corresponding line. Generally, the awk, sed, tr magic seems a bit convoluted, but there's probably no alternative. I have not tested this workflow, but trust that you've done your tests of it.

Thanks. Testing workflows is a bit of a pain. I have tested the issue and PR creation on this branch. Scheduled workflows only run on `main` so we will know after merging if everything runs as expected.

**Review by coquelin77 — APPROVED** [review]

**Comment by coquelin77** [comment]
rerun tests

**Comment by coquelin77** [comment]
rerun tests

**Pull request merged**